lib/tasks/clinicard_process_ssrs_move.rake

```diff
@@ -0,0 +1,85 @@
+task clincard_process_ssrs_move: :environment do
```

Comment:
Is this something that can be made into a generalized task where we pass the organization ID to it? This task will only work for an MUSC instance of SPARCRequest and writing a new task for making changes to an organization individually seems redundant when we could have a single task that can be re-used.

Reply:
I'll rename it something more generic. You're right, it can be used for other organizations as well. I'll refactor to accept the id as an argument.